PayPal.xcodeproj/xcshareddata/xcschemes/PayPal.xcscheme

```diff
@@ -14,7 +14,7 @@
             buildForAnalyzing = "YES">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "PayPal::PayPal"
+               BlueprintIdentifier = "80DB6F0D26B89D9600277E54"
```

Comment:
quick question, what is this ID, is it auto generated?

Reply:
This is auto generated - Looking through why this change would happen here led me to [this](https://github.com/tuist/XcodeProj/pull/612#pullrequestreview-672012032). Basically, I think that the previous value (`PayPal::PayPal`) is not a UUID, and Xcode generated an actual identifier for the scheme when I was looking at all the schemes through the editor.

 @scannillo - did you by chance use `swift package generate-xcodeproj` to create the project initially?

Reply:
I definitely did use that command for part of the setup, yup!

Reply:
ah that makes sense!!! Thanks for explaining Jon!! I'll resolve these 2 comments :smile: